behave/features/steps/specialist_provider.py

```diff
@@ -366,5 +370,67 @@ def step_impl(context):
     assert CASE_SPLIT_TEXT in full_case_log, "Could not find split case in operators dashboard"
 
 
+@step(u'I am on the CSV upload page')
+def step_impl(context):
+    csv_page = f"{CLA_FRONTEND_URL}/provider/csvupload/"
+    context.helperfunc.open(csv_page)
+    page = f"/provider/csvupload/"
+    wait_until_page_is_loaded(page, context)
+    header = context.helperfunc.find_by_xpath("//header/h2[text()='CSV Upload']")
+    assert header is not None
+
+
+@step(u'I select \'Choose file\' and upload a csv file')
+def step_impl(context):
+    input_csv = context.helperfunc.find_by_xpath("//input[@name='csvfile']")
+    # test will fail if file can't be found or upload fails.
+    input_csv.send_keys(os.getcwd() + "/uploads/csvupload.csv")
```

Comment:
can we have this file path specified in constants rather than in code?

Reply:
Now in constants